signer/src/config/mod.rs

```diff
@@ -756,17 +755,39 @@ mod tests {
     fn default_config_toml_loads_signer_private_key_config_with_environment() {
         clear_env();
 
-        let new = "a1a6fcf2de80dcde3e0e4251eae8c69adf57b88613b2dcb79332cc325fa439bd";
+        let new = "9bfecf16c9c12792589dd2b843f850d5b89b81a04f8ab91c083bdf6709fbefee01";
```

Comment:
Why add those two bytes at the end here?

Reply:
Oh, you used the private key associated with one of the keys in the bootstrap signing set so that these tests pass. Still, we should lop off the byte, it is a nonstandard private key representation.

Reply:
I'm stripping two bytes programmatically. To advocate for what I have now, current key can be searched and found in some of our readme files, describing what this key is, but I don't mind to delete this bytes manually